lib/scholar/metrics/similarity.ex

```diff
@@ -34,29 +53,59 @@ defmodule Scholar.Metrics.Similarity do
         0.25
       >
 
+      iex> x = Nx.iota({2,3})
```

Comment:
I think if you write the tensor explicitly it's easier to visualize the calculation

Reply:
Ok

Reply:
```suggestion
      iex> x = Nx.tensor([[0, 1, 2], [3, 4, 5]])
```